server/service/src/sync/program_schemas/hiv_care_encounter_ui_schema.json

```diff
@@ -663,13 +665,18 @@
           "type": "Spacer"
         },
         {
-          "type": "EncounterProgramEvent",
+          "type": "ProgramEvent",
           "scope": "#/properties/arvMedication/properties/quantityPrescribed",
           "label": "Pill Count (at last visit end)",
           "options": {
             "eventType": "arvMedicationQuantityPrescribed",
+            "documentType": "HIVCareEncounter",
             "display": {
               "type": "number"
+            },
```

Comment:
Was always returning 9 from the latest encounter instead of the correct value 👀

Reply:
Is this still a problem? it sounds like a problem I had with the demo data and how the data is populated (the timestamps)... will have a look later if its not working probably...

Reply:
Not a problem anymore. It's because schema wasn't specifying "at"